### ISSUE TITLE
Parameter scenario

### ISSUE THREAD

**Issue by RBSJonker** [LCA-ActivityBrowser/activity-browser#1300]
### Issue

I conducted a preliminary test by adjusting the pumping efficiency through multiplying the electricity input. However, the LCA results did not show any differences, contrary to my expectations. Please see the attached images for the activity details, Excel scenarios, and LCA outcomes.

![image](https://github.com/LCA-ActivityBrowser/activity-browser/assets/49594068/d8679676-ab65-4151-96a6-088cbfd1430e)

![image](https://github.com/LCA-ActivityBrowser/activity-browser/assets/49594068/01357422-f64b-4717-88c9-6634148b7fd6)

![image](https://github.com/LCA-ActivityBrowser/activity-browser/assets/49594068/43933b26-d81e-4b96-97f4-d217c7635d26)

**Comment by mrvisscher** [comment]
Hi @RBSJonker,

Could you please supply your version of Activity Browser and the output of the `conda list` command for the AB conda environment?

Thanks!

Kind regards,

Mairn

**Comment by mrvisscher** [comment]
Hi @RBSJonker,

Looking closer at your results I see that you've used the electricity process in your functional unit, which on its own doesn't change for your test run. Please try to rerun the Scenario LCA with the `Pumping` activity as FU and see if it gives the desired results.

**Comment by RBSJonker** [comment]
If I import the scenario for only parameter eta_1 I get this: 

![image](https://github.com/LCA-ActivityBrowser/activity-browser/assets/49594068/431dbfbb-8a0f-4557-8fd7-516cdb12852a)
and it should be this

![image](https://github.com/LCA-ActivityBrowser/activity-browser/assets/49594068/b3180afb-311a-4087-aaea-eac6753133b9)

**Comment by mrvisscher** [comment]
Did you try pressing `Reset table` yet to see whether that updates the results to the expected parameter values?

**Comment by RBSJonker** [comment]
Yes I press that and it deleted all columns, default, high eff, medium eff etc

**Comment by RBSJonker** [comment]
I solved it but I am not sure what the issue was. I did the following: 

Gave the group name the same as group name in AB. No idea how I can set the group name in AB but this what I did.

**Comment by RBSJonker** [comment]
Now it does not load anymore. This is my CMD output: 

Loading Scenario file. This may take a while for large files
Superstructure: Attempting to read as parameter scenario file.
<function ScenarioImportWidget.scenario_db_check at 0x0000026C15D86200> -- 0.0009582042694091797
<function SuperstructureManager.fill_empty_process_keys_in_exchanges at 0x0000026C159374C0> -- 0.0009584426879882812
<function SuperstructureManager.verify_scenario_process_keys at 0x0000026C15937600> -- 0.0009953975677490234
<function SuperstructureManager.check_duplicates at 0x0000026C15937880> -- 0.0
<function ScenarioImportWidget.sync_superstructure at 0x0000026C15D860C0> -- 0.007943153381347656
<function ScenarioImportWidget.load_action at 0x0000026C15D85F80> -- 19.50167894363403
